tests/test_checker.py

```diff
@@ -30,6 +30,7 @@
 import checker
 
 TEST_MANIFEST = os.path.join(tests_dir, 'org.externaldatachecker.Manifest.json')
+CVE_TEST_MANIFEST = os.path.join(tests_dir, 'org.cvechecker.Manifest.json')
```

Comment:
Hmm, this way other checkers don't run your manifest. Maybe it'd be better to add your tests to the big manifest?